js/ui/scheduler/utils.js

```diff
@@ -0,0 +1,21 @@
+import dateUtils from '../../core/utils/date';
+
+const toMs = dateUtils.dateToMilliseconds;
+
+const getTimezoneChangeDurationInMinutes = (startDate, endDate, updatedStartDate, updatedEndDate) => {
```

Comment:
The name of the method does not reflect its essence. We calculate the duration of the appointment when switching to summer/winter time.